docs/howto/manage-charms.md

```diff
@@ -11,6 +11,57 @@ Use Charmcraft to quickly initialise your charm project. This generates the
 folder structure, creates placeholder configuration and code files, and
 configures development tooling.
 
+The Charmcraft profiles also all provide the recommended tooling and
+configuration for developing charms. You'll need to install
+[`tox`](https://tox.wiki/en/stable/), and can then run ``tox list`` in the root
+folder to see the available commands.
+
+```{admonition} Best practice
+:class: hint
+
+The quality assurance pipeline of a charm should be automated using a
+continuous integration (CI) system.
+```
+
+```{tip}
+To prepare an environment for running integration tests, such as in CI, use
+the `concierge tool <https://github.com/jnsgruk/concierge>`_ or the
+`actions-operator <https://github.com/charmed-kubernetes/actions-operator>`_.
+```
+
+```{tip}
+The `charming-actions <https://github.com/canonical/charming-actions>`_
+repository includes actions to ensure that libraries are up-to-date, publish
+charms and libraries, and more.
+```
+
+## Use the provided tooling to maintain style and detect issues early
+
+The Charmcraft profile you've chosen has configured a number of recommended
+tools for developing charms. To use these, [install
+`tox`](https://tox.wiki/en/stable/installation.html#as-tool) on your development
+server.
+
+```{tip}
```

Comment:
What about instead of a separate Tip box (I really find that so many of them distracts from the text):

```
To use these [install tox](...) on your development server
(it's already installed if you're using Multipass or concierge).
```